tests/test_enterprise/api/test_filters.py

```diff
@@ -54,7 +54,7 @@ def test_filter_for_list(self, is_staff, expected_data_length):
         assert len(data['results']) == expected_data_length
 
     @ddt.data(
-        (False, False, {'detail': 'Not found.'}),
+        (False, False, {'detail': 'No EnterpriseCustomer matches the given query.'}),
```

Comment:
All of these changes are the result of the new DRF library being more explicit about 404 errors. There should be no impact because consumers shouldn't really use this message string to drive business logic.